doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -307,6 +307,7 @@ nRF5340 Audio
   * The documentation for :ref:`nrf53_audio_app_building` with cross-links and additional information.
   * The :file:`buildprog.py` is an app-specific script for building and programming multiple kits and cores with various audio application configurations. The script will be deprecated in a future release. The audio applications will gradually shift only to using standard tools for building and programming development kits.
   * The :ref:`nrf53_audio_app` :ref:`nrf53_audio_app_building_script` now builds into a directory for each transport, device type, core, and version combination.
+  * Will now use the NFC.TAGHEADER0 value from FICR as the broadcast ID instead of using a random ID.
```

Comment:
```suggestion
  * The application will now use the ``NFC.TAGHEADER0`` value from FICR as the broadcast ID instead of using a random ID.
```
Please fix the text.

Reply:
Done